debezium-core/src/main/java/io/debezium/time/MicroDuration.java

```diff
@@ -8,26 +8,25 @@
 
 import java.math.BigDecimal;
 import java.time.temporal.ChronoUnit;
+
 import org.apache.kafka.connect.data.Schema;
 import org.apache.kafka.connect.data.SchemaBuilder;
 
 /**
- * A utility representing a duration into a corresponding {@link SchemaBuilder#float64() FLOAT64} 
+ * A utility representing a duration into a corresponding {@link SchemaBuilder#int64() INT64}
  * number of <em>microsecond</em>, and for defining a Kafka Connect {@link Schema} for duration values.
- * 
- * The duration is amount is stored in floating point as opposed to an integer because certain DBs (e.g. Postgres) will store
```

Comment:
@jpechane, this comment makes the original motivation for using float clear: it's indeed the larger value range. I'm still inclined to prefer the int64 mapping, as the fractional part just doesn't make sense. In that light, we may do the String-based mapping ([DBZ-1498](https://issues.jboss.org/browse/DBZ-1498) sooner than later, though (e.g. "1Y2M3DT4H5M6.78S").